scripts/tf_cnn_benchmarks/benchmark_cnn.py

```diff
@@ -1353,7 +1360,9 @@ def __init__(self, params, dataset=None, model=None):
     if self.params.variable_update == 'horovod' and self.params.job_name:
       raise ValueError('job_name should not be specified for Horovod.')
 
-    if self.params.use_fp16 and self.params.fp16_enable_auto_loss_scale:
+    self.enable_auto_loss_scale = self.params.auto_mixed_precision or (
```

Comment:
Instead of automatically using auto loss scaling, I would use the same loss scale settings as if `--use_fp16` were used, which uses a static loss scale by default. I think this just requires modifying `_maybe_initialize_fp16` to create the loss scale if `auto_mixed_precision` is used as well.